www/cgit/Portfile

```diff
@@ -88,7 +91,11 @@ livecheck.type  regex
 livecheck.url   ${homepage}log/?h=master
 livecheck.regex {>v([0-9.]+)<}
 
-notes "
-A sample config file have been placed in:
-  » ${prefix}/etc/cgitrc.sample
-"
+post-activate {
+    ui_msg "
+::
+:: A sample config file have been placed in:
+::   ${prefix}/etc/cgitrc.sample
+::
+    "
+}
```

Comment:
Please revert this change. `notes` is the correct way to display post-installation notes to the user.